pkg/sloop/webserver/webfiles/index.html

```diff
@@ -36,11 +36,13 @@
             <select name="query" id="filterquery" hidden="true"> </select>
             <!-- <br><br> -->
 
+            <!-- Temporarily disable Endtime (Workaround for W-9423151) 
```

Comment:
As this is UI change you would need to update binddata.go file too

Reply:
Updated webfiles folder